nixos/modules/security/pam_mount.nix

```diff
@@ -29,6 +29,26 @@ in
           xlink:href="http://pam-mount.sourceforge.net/pam_mount.conf.5.html" />.
         '';
       };
+
+      additionalSearchPaths = mkOption {
+        type = types.listOf types.package;
+        default = [];
+        example = "[ pkgs.bindfs ]";
```

Comment:
Not fully sure but shouldn't this be using `literalExample`?

Reply:
Fixed it